README.md

```diff
@@ -144,7 +144,7 @@ jobs:
       - name: actions-template-sync
         uses: AndreasAugustin/actions-template-sync@v2
         with:
-          org_github_token: ${{ steps.generate_token.outputs.token }}
```

Comment:
Totally makes sense to change the name. But this needs a MAJOR version change. For this change/fix I want to stay within the current version.

Reply:
Sounds reasonable 👍